src/js/storage.js

```diff
@@ -244,6 +252,9 @@ BadgerPen.prototype = {
    */
   setupHeuristicAction: function(domain, action){
     this._setupDomainAction(domain, action, "heuristicAction");
+    if (this.isActionMapEmptyForDomain(domain)) {
+      this.getBadgerStorageObject('action_map').deleteItem(domain);
+    }
```

Comment:
Ah, good thinking. Best not have to end up with another migration to fix the same issue in the future :)